credsweeper/filters/value_token_base32_check.py

```diff
@@ -0,0 +1,24 @@
+from password_strength import PasswordStats
+
+from credsweeper.credentials import LineData
+from credsweeper.filters import Filter
+
+
+class ValueTokenBase32Check(Filter):
+    """Check that candidate have good randomization"""
+
+    def run(self, line_data: LineData) -> bool:
+        """Run filter checks on received credential candidate data 'line_data'.
+
+        Args:
+            line_data: credential candidate data
+
+        Return:
+            True, if need to filter candidate and False if left
+
+        """
+        if not line_data.value:
+            return True
+
+        stats = PasswordStats(line_data.value)
+        return bool(0.7 > stats.strength())
```

Comment:
So it means `password_strength` measure the strength with in 0 ~ 1.0?
Then why you set pass condition as `0.7`? Could you explain it more?

Reply:
Tokens looked like BASE32 encoded data have only 16 characters length. 
An entropy will be weak anyway.
The "strength" calculates something like "data complexity". 
0.7 was chosen manually with false positive cases review.

Reply:
As I told, please make the number to const.